src/storagesync/azext_storagesync/_help.py

```diff
@@ -0,0 +1,289 @@
+               "SampleSyncGroup" --name "SampleServerEndpoint"
+"""
+
+helps['storagesync server-endpoint list'] = """
+    type: command
+    short-summary: List all server endpoints in a sync group.
+    examples:
+      - name: List all server endpoints in sync group "SampleSyncGroup".
+        text: |-
+               az storagesync server-endpoint list --resource-group "SampleResourceGroup" \\
+               --storage-sync-service-name "SampleStorageSyncService" --sync-group-name \\
+               "SampleSyncGroup"
+"""
+
+helps['storagesync server-endpoint wait'] = """
+    type: command
+    short-summary: Place the CLI in a waiting state until a condition of a server endpoint is met.
+    examples:
+      - name: Place the CLI in a waiting state until a condition of a server endpoint is created.
+        text: |-
+               az storagesync server-endpoint wait --resource-group "SampleResourceGroup" \\
+               --storage-sync-service-name "SampleStorageSyncService" --sync-group-name \\
+               "SampleSyncGroup" --name "SampleServerEndpoint" --created
+"""
+
+helps['storagesync registered-server'] = """
+    type: group
+    short-summary: Manage registered server.
+"""
+
+helps['storagesync registered-server delete'] = """
+    type: command
+    short-summary: Unregister an on-premises server from it's storage sync service.
+    long-summary: Unregister an on-premises server from it's storage sync service which will result in cascading deletes of all server endpoints on this server.
+    examples:
+      - name: Unregister an on-premises server "41166691-ab03-43e9-ab3e-0330eda162ac" from it's storage sync service "SampleStorageSyncService".
+        text: |-
+               az storagesync registered-server delete --resource-group "SampleResourceGroup" --storage-sync-service-name \\
+               "SampleStorageSyncService" --server-id "41166691-ab03-43e9-ab3e-0330eda162ac"
+"""
+
+helps['storagesync registered-server show'] = """
+    type: command
+    short-summary: Show the properties for a given registered server.
+    examples:
+      - name: Show the properties for registered server "080d4133-bdb5-40a0-96a0-71a6057bfe9a".
+        text: |-
+               az storagesync registered-server show --resource-group "SampleResourceGroup" --storage-sync-service-name \\
+               "SampleStorageSyncService" --server-id "080d4133-bdb5-40a0-96a0-71a6057bfe9a"
+"""
+
+helps['storagesync registered-server list'] = """
+    type: command
+    short-summary: List all registered servers for a given storage sync service.
+    examples:
+      - name: List all registered servers for storage sync service "SampleStorageSyncService".
+        text: |-
+               az storagesync registered-server list --resource-group "SampleResourceGroup" --storage-sync-service-name \\
+               "SampleStorageSyncService"
+"""
+
+helps['storagesync registered-server wait'] = """
```

Comment:
pair with no-wait?

Reply:
Yes, because we define 'delete' as supporting 'no-wait', we need a 'wait' command to query the 'delete' status.

---
In reply to: [392648677](https://github.com/Azure/azure-cli-extensions/pull/1389#discussion_r392648677) [](ancestors = 392648677)